carbon-location.html

```diff
@@ -148,11 +148,18 @@
         }
       },
 
+      ready: function() {
+        this._isReady = true;
```

Comment:
😓 I misspelled it when I searched.  We've shipped a use of it in paper-dialog-behavior.